tools/git_lfs.rst

```diff
@@ -34,6 +34,19 @@ Git LFS requires Git version 1.8.2 or later to be installed.
 
 Before you can use Git LFS with LSST data you'll need to configure by following the next section.
 
+.. _git-lfs-basic-config:
+
+Basic Git LFS configuration
```

Comment:
I like the single "Configuring Git LFS" section with subsections. I'll go with that. :+1: